frontend/javascripts/oxalis/view/statusbar.tsx

```diff
@@ -491,40 +479,89 @@ function Infos() {
           />
         </span>
       ) : null}
-      {representativeResolution && (
-        <span className="info-element">
-          <img
-            src="/assets/images/icon-statusbar-downsampling.svg"
-            className="resolution-status-bar-icon"
-            alt="Resolution"
-          />{" "}
-          <Tooltip
-            title={
-              <>
-                Rendered magnification per layer:
-                <ul>
-                  {Object.entries(activeMagOfEnabledLayers).map(([layerName, mag]) => {
-                    const readableName = getReadableNameForLayerName(dataset, tracing, layerName);
-
-                    return (
-                      <li key={layerName}>
-                        {readableName}: {mag ? mag.join("-") : "none"}
-                      </li>
-                    );
-                  })}
-                </ul>
-              </>
-            }
-          >
-            {representativeResolution.join("-")}
-            {isActiveResolutionGlobal ? "" : "*"}{" "}
-          </Tooltip>
-        </span>
-      )}
+      <ResolutionInfo />
     </React.Fragment>
   );
 }
 
+function DownloadSpeedometer() {
+  const [currentBucketDownloadSpeed, setCurrentBucketDownloadSpeed] = useState<number>(0);
+  const [totalDownloadedByteCount, setTotalDownloadedByteCount] = useState<number>(0);
+  useInterval(() => {
+    const { avgDownloadSpeedInBytesPerS, accumulatedDownloadedBytes } =
+      getGlobalDataConnectionInfo().getStatistics();
+    setCurrentBucketDownloadSpeed(avgDownloadSpeedInBytesPerS);
+    setTotalDownloadedByteCount(accumulatedDownloadedBytes);
+  }, 1500);
+
+  return (
+    <FastTooltip
+      title={`Downloaded ${formatCountToDataAmountUnit(
+        totalDownloadedByteCount,
+      )} of Image Data (after decompression)`}
+    >
+      <DownloadOutlined className="icon-margin-right" />
+      {formatCountToDataAmountUnit(currentBucketDownloadSpeed)}/s
+    </FastTooltip>
+  );
+}
+
+function ResolutionInfo() {
+  const { representativeResolution, isActiveResolutionGlobal } =
+    useSelector(getActiveResolutionInfo);
+  const [magTooltipContent, setMagTooltipContent] = useState<string | null>(null);
+
+  const renderMagTooltipContent = useCallback(() => {
+    const state = Store.getState();
+    const { activeMagOfEnabledLayers } = getActiveResolutionInfo(state);
```

Comment:
Ok it seems to work perfectly fine. Do you know why this works? Becauase of the 
`const { representativeResolution, isActiveResolutionGlobal } = useSelector(getActiveResolutionInfo);`
selector maybe? This forces the `ResolutionInfo` to rerender in case the resolution / mag changes. But what why is the `renderMagTooltipContent` not cached / returning the updated value? 

=> Ok found the answer myself:
`useCallback` caches the method definition itself and not the return value like `useMemo`. :)